internal/testrunner/reporters/outputs/file.go

```diff
@@ -65,3 +56,28 @@ func testReportsDir() (string, error) {
 	}
 	return filepath.Join(buildDir, "test-results"), nil
 }
+
+// CleanTestReportsDir removes creates the test reports folder if it already exists, then
+// creates it, effectively resulting in a clean (aka empty) test reports folder.
+func CleanTestReportsDir() error {
```

Comment:
I'm OK with the logic, although I'd like to know how the reports are generated. If the generation process always writes the reports to the proper place, creating or overwriting the content, then I wonder if the clean one should throw an error (ignoring them if they occur).

Again, my concern is already covered by this PR, so please fee free to merge and discuss in a follow-up not to block it.

Thanks!

Reply:
Yeah, lets move forward with this PR and then address any more concerns as they come up when we start using this more.